packages/iTwinUI-react/src/core/utils/components/Resizer.tsx

```diff
@@ -0,0 +1,298 @@
+        }
+        case 'top-right': {
+          const newHeight = initialHeight + (initialPointerY - clientY);
+          if (newHeight >= minHeight) {
+            height = elementRef.current.style.height = `${newHeight}px`;
+            translateY = initialTranslateY - (initialPointerY - clientY);
+          }
+          width = elementRef.current.style.width = `${
+            initialWidth - (initialPointerX - clientX)
+          }px`;
+          elementRef.current.style.transform = `translate(${translateX}px, ${translateY}px)`;
+          break;
+        }
+        case 'right': {
+          width = elementRef.current.style.width = `${
+            initialWidth - (initialPointerX - clientX)
+          }px`;
+          break;
+        }
+        case 'bottom-right': {
+          width = elementRef.current.style.width = `${
+            initialWidth - (initialPointerX - clientX)
+          }px`;
+          height = elementRef.current.style.height = `${
+            initialHeight - (initialPointerY - clientY)
+          }px`;
+          break;
+        }
+        case 'bottom': {
+          height = elementRef.current.style.height = `${
+            initialHeight - (initialPointerY - clientY)
+          }px`;
+          break;
+        }
+        case 'bottom-left': {
+          const newWidth = initialWidth + (initialPointerX - clientX);
+          if (newWidth >= minWidth) {
+            width = elementRef.current.style.width = `${newWidth}px`;
+            translateX = initialTranslateX - (initialPointerX - clientX);
+          }
+          height = elementRef.current.style.height = `${
+            initialHeight - (initialPointerY - clientY)
+          }px`;
+          elementRef.current.style.transform = `translate(${translateX}px, ${translateY}px)`;
+          break;
+        }
+        case 'left': {
+          const newWidth = initialWidth + (initialPointerX - clientX);
+          if (newWidth < minWidth) {
+            break;
+          }
+          width = elementRef.current.style.width = `${newWidth}px`;
+          translateX = initialTranslateX - (initialPointerX - clientX);
+          elementRef.current.style.transform = `translate(${translateX}px, ${translateY}px)`;
+          break;
+        }
+        default:
+          break;
+      }
+    };
+
+    elementRef.current.ownerDocument.addEventListener(
+      'pointermove',
+      onResizePointerMove,
+    );
+    elementRef.current.ownerDocument.addEventListener(
+      'pointerup',
+      () => {
+        document.removeEventListener('pointermove', onResizePointerMove);
+        if (elementRef.current) {
+          elementRef.current.ownerDocument.body.style.userSelect = originalUserSelect;
+          setStyle?.({
+            width,
+            height,
+            transform: `translate(${translateX}px, ${translateY}px)`,
+          });
+        }
+      },
+      { once: true },
+    );
+  };
+
+  return (
+    <>
+      <div
+        data-iui-resizer='top-left'
```

Comment:
> but now I'm thinking we _should_ have styling for these like a colored border when hovering edges.

I guess question for @FlyersPh9

Reply:
Can we try releasing without coloring the edges and see if we get any feedback on users not knowing it can be resized?  I'm not used to seeing dialogs where the edges / corners light up to show that they can be resized.